sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -9,12 +9,72 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     this.listenTo(this.categories, 'sync change:unread_count', this.render);
   },
 
+  events: {
+    'click #login-submit': 'loginSubmit',
+  },
+
+  loginSubmit: function(e){
+    e.preventDefault()
+
+    var formData = JSON.stringify($("#login").serializeObject());
+
+    $.ajax({
+      type: "POST",
+      url: "/login",
+      data: formData,
+      success: function(response){
+        var user = response.response.user;
+        if (user) {
+          location.reload();
+        }
+        var errors = response.response.errors;
+        if (errors) {
+          errorMsg = errors.email || errors.password || errors.rememeber || {};
```

Comment:
I agree that we won't, but Flask-Security can return errors on remember, so there's no reason not to catch them here since it costs us nothing.